src/Components/Web.JS/babel.config.json

```diff
@@ -0,0 +1,3 @@
+{
+    "presets": [ [ "@babel/preset-env", { "targets": { "node": true } } ] ]
+}
```

Comment:
Since [7.13.0 babel/core version](https://babeljs.io/blog/2021/02/22/7.13.0#top-level-targets-option-12189httpsgithubcombabelbabelpull12189-rfchttpsgithubcombabelrfcspull2), the config could be written this way:

`{
"targets": { "node": true },
"presets": ["@babel/preset-env"]  
}`

Reply:
you could use the top-level targets option new feature for babel/core since version 7.13.0.